lib/shopify_app/login_protection.rb

```diff
@@ -26,6 +26,7 @@ def shop_session
 
     def login_again_if_different_shop
       if shop_session && params[:shop] && params[:shop].is_a?(String) && shop_session.url != params[:shop]
+        session[:shopify] = nil
```

Comment:
I added it so that when error handlers (like errbit) dumped the session we had the shop domain not just the id. We had an issue you made for this